product.rb

```diff
@@ -0,0 +1,46 @@
+# Info's product
+#
+# Including info about:
+# sale_tax consist import tax and good tax
+# price
+# price_with_sale_tax
+
+class Product
+
+  IMPORTED_PRODUCT = %w(imported)
+
+  def self.imported?(item)
+    product = item[:product].split(" ")
+    combine_product = (IMPORTED_PRODUCT + product)
+    filter_product = combine_product.uniq
+    combine_product.length != filter_product.length
+  end
+
```

Comment:
why not initialize product with an item and call production instance to see whether it is ```imported?``` or ```exempt```?